tools/StaticAnalysis/Exceptions/SignatureIssues.csv

```diff
@@ -1087,3 +1087,11 @@
 "Microsoft.Azure.Commands.AnalysisServices.dll","Microsoft.Azure.Commands.AnalysisServices.NewAzureRmAnalysisServicesFirewallRule","New-AzureRmAnalysisServicesFirewallRule","1","8100","New-AzureRmAnalysisServicesFirewallRule Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.Commands.AnalysisServices.dll","Microsoft.Azure.Commands.AnalysisServices.NewAzureRmAnalysisServicesFirewallConfig","New-AzureRmAnalysisServicesFirewallConfig","1","8100","New-AzureRmAnalysisServicesFirewallConfig Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.WindowsAzure.Commands.Storage.dll","Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet.EnableAzureStorageServiceDeleteRetentionPolicyCommand","Enable-AzureStorageDeleteRetentionPolicy","1","8410","Parameter RetentionDays of cmdlet Enable-AzureStorageDeleteRetentionPolicy does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.Commands.Dns.dll","Microsoft.Azure.Commands.Dns.SetAzureDnsZone","Set-AzureRmDnsZone","1","8410","Parameter RegistrationVirtualNetworkIds of cmdlet Set-AzureRmDnsZone does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
Please change all of the "VirtualNetworkIds" to "VirtualNetworkId" and all of the "VirtualNetworks" to "VirtualNetwork".  This is a powershell standard.  Once you do that you can remove these lines from SignatureIssues.